contrib/ibmock/lib.h

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright (c) 2024, NVIDIA CORPORATION. All rights reserved.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifndef __LIB_H
+#define __LIB_H
+#include <assert.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <pthread.h>
+#include <sys/uio.h>
+
+
+struct array {
+    void   *data;
+    size_t elem_size;
+    int    count;
+};
+
+pthread_mutex_t global_lock = PTHREAD_MUTEX_INITIALIZER;
```

Comment:
is it a safe way to declare and define global variable?
What if it's included by multiple translation units?
I think we need `extern` here, and define in C file

Reply:
that's a miss while clean-up, moved to C file thanks